source/common/network/utility.cc

```diff
@@ -235,4 +234,34 @@ bool Utility::getOriginalDst(int fd, sockaddr_storage* orig_addr) {
   return (status == 0);
 }
 
+void Utility::parsePortRangeList(const std::string& string, std::list<PortRange>& list) {
+  std::vector<std::string> ranges = StringUtil::split(string.c_str(), ',');
+  for (const std::string& s : ranges) {
+    std::stringstream ss(s);
+    uint32_t min = 0;
+    uint32_t max = 0;
+
+    if (s.find('-') != std::string::npos) {
+      char dash = 0;
+      ss >> min;
```

Comment:
Don't we need some error checking in here somewhere in case what is passed isn't all numbers, commas, and dashes in correct sequence?